tests/testthat/test-eda.R

```diff
@@ -1,3 +1,56 @@
-test_that("multiplication works", {
-  expect_equal(2 * 2, 4)
-})
+test_describe_num_var <- function() {
```

Comment:
Add a docstring for the function

Reply:
I'm not sure a docstring is necessary for the test function. I added a very short one to show which function it tests.